suite-native/module-send/src/screens/SendOutputsReviewScreen.tsx

```diff
@@ -34,25 +33,26 @@ export const SendOutputsReviewScreen = ({
 }: StackProps<SendStackParamList, SendStackRoutes.SendOutputsReview>) => {
     const { accountKey } = route.params;
     const { translate } = useTranslate();
-    const dispatch = useDispatch();
     const navigation = useNavigation<NavigationProps>();
+    const showReviewCancellationAlert = useShowReviewCancellationAlert();
+    const dispatch = useDispatch();
 
     useEffect(() => {
         const unsubscribe = navigation.addListener('beforeRemove', async e => {
-            if (e.data.action.type === 'GO_BACK') {
-                e.preventDefault();
-            }
-            const response = await dispatch(cancelSignSendFormTransactionThunk());
-            if (isFulfilled(response)) {
-                // If success navigation is handled by signTransactionThunk call on SendAddressReviewScreen.
+            // We want to modify only behavior of back button actions.
+            if (e.data.action.type !== 'GO_BACK') return;
```

Comment:
maybe you can add `gestureEnabled: false` to these screens so user cant go back with swipe on iOS (I know this is for the future, but for sure). Wdyt?

Reply:
Good thinking. Added here: [suite-native/module-send/src/navigation/SendStackNavigator.tsx](https://github.com/trezor/trezor-suite/pull/14540/files#diff-5fb2379ca96eb38894f5599ee32afd7eb7f558e1aaf22a59b8821b8ce48328c4)